test/OpenTelemetry.Tests.Stress/Skeleton.cs

```diff
@@ -60,13 +60,20 @@ public static void Stress(int concurrency = 0, int prometheusPort = 0)
             description: "The rate of `Run()` invocations based on a small sliding window of few hundreds of milliseconds.");
         var dCpuCyclesPerLoop = 0D;
 
+#if NETFRAMEWORK
```

Comment:
unrelated to the PR scope?

Reply:
Some errors/warnings were detected by the build system after the version bump.